src/users/users.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from '@nestjs/common';
+import { UsersService } from './users.service';
+import { UsersResolver } from './users.resolver';
+import { SequelizeModule } from '@nestjs/sequelize';
+import { User } from './entities/user.entity';
+import { UsersRepository } from './repositories/users.repository';
```

Comment:
Codacy has a fix for the issue: [Insert `';⏎import·{·UsersResolver·}·from·'./users.resolver';⏎import·{·UsersService·}·from·'./users.service`](https://app.codacy.com/gh/uigormarshall/back-challenge-graphql/pullRequest?prid=8283306)
```suggestion
import { UsersRepository } from './repositories/users.repository';
```